crates/jstz_cli/src/logs/mod.rs

```diff
@@ -9,36 +11,29 @@ mod trace;
 pub enum Command {
     /// View logs
     Trace {
-        /// View console.log messages
-        #[arg(long)]
-        log: bool,
-        /// View console.info messages
-        #[arg(long)]
-        info: bool,
-        /// View console.warn messages
-        #[arg(long)]
-        warn: bool,
-        /// View console.error messages
-        #[arg(long)]
-        error: bool,
-        /// View contract creations
-        #[arg(long)]
-        contract: bool,
-        /// Add custom search strings
-        #[arg(long, value_name = "custom_strings")]
-        custom: Vec<String>,
+        // The address or the alias of the smart function
+        #[arg(value_name = "ALIAS|ADDRESS")]
+        smart_function: String,
     },
 }
 
-pub fn exec(command: Command, cfg: &mut Config) -> Result<()> {
+pub async fn exec(command: Command, cfg: &mut Config) -> Result<()> {
     match command {
-        Command::Trace {
-            log,
-            info,
-            warn,
-            error,
-            contract,
-            custom,
-        } => trace::exec(log, info, warn, error, contract, custom, cfg),
+        Command::Trace { smart_function } => {
+            let address = get_address_for_smart_function(&smart_function, cfg)?;
+            trace::exec(address, cfg).await
```

Comment:
Nit: to match the style of other commands, it would be better to have just trace::exec(address, cfg).await here that does 'let address  = ...' parsing inside of it, similarly to main.rs.